src/state-summary/state-card-climate.html

```diff
@@ -52,14 +56,23 @@
       type: Boolean,
       value: false,
     },
-
     stateObj: {
       type: Object,
     },
   },
 
   computeTargetTemperature: function (stateObj) {
-    return stateObj.attributes.temperature + ' ' + stateObj.attributes.unit_of_measurement;
+    if (stateObj.attributes.operation_mode.includes('Auto')) {
```

Comment:
@balloob It looks like `indexOf` can be used with strings and not just arrays. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf